core/src/main/java/org/kabeja/math/Extrusion.java

```diff
@@ -116,4 +116,53 @@ public Vector getNormal() {
     public Vector getDirectionZ() {
         return n;
     }
+
+    /**
+     * Transforms a point from WCS to OCS.
+     * Relies on the arbitrary axis algorithm to determine the OCS.
+     * @param p point in WCS
+     * @return point in OCS
+     */
+    public Point3D wcsToOcs(Point3D p) {
+        Vector ox = getDirectionX();
+        Vector oy = getDirectionY();
+        Vector oz = n;
```

Comment:
@JohannesMessner No need to create a new variable oz, unless it's for readability purposes. But I don't know what o means anyway... nor n actually. I had to go up and read n means normalVector. o for ocsX??